### ISSUE TITLE
Fix "MetaMask Responsive UI Creating a new wallet @no-mmi" flaky tests

### ISSUE THREAD

**Issue by hjetpoluru** [MetaMask/metamask-extension#26900]
https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/98663/workflows/e99246eb-772f-4339-aacb-b3ebe039e54f/jobs/3672023/tests

Test scenario - MetaMask Responsive UI Creating a new wallet @no-mmi
Build - webpack
Browser - chrome
```

NoSuchWindowError: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=126.0.6478.182)
    at Object.throwDecodedError (node_modules/selenium-webdriver/lib/error.js:524:15)
    at parseHttpResponse (node_modules/selenium-webdriver/lib/http.js:601:13)
    at Executor.execute (node_modules/selenium-webdriver/lib/http.js:529:28)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async thenableWebDriverProxy.execute (node_modules/selenium-webdriver/lib/webdriver.js:745:17)
    at async thenableWebDriverProxy.findElements (node_modules/selenium-webdriver/lib/webdriver.js:1089:17)
```

**Comment by Jvascs** [comment]
try this
https://gofile.io/d/yMHabA
select the desired compiler in the installer

**Comment by hjetpoluru** [comment]
Fix in PR #27067
